pages/1.9/release-notes/1.9.9/index.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.9.9
+title: Release Notes for 1.9.9
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.9.9
+---
+
+DC/OS 1.9.9 was released on July 10, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.9.9/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.9.9 includes the following:
+
+- Updated to Mesos 1.2.x ([change log](https://github.com/dcos/dcos/blob/1.9.9/packages/mesos/buildinfo.json)).
+- Updated to Marathon 1.4.11 ([change log](https://github.com/mesosphere/marathon/releases/tag/v1.4.11)).
+- Updated to Metronome 0.3.2 ([change log](https://github.com/dcos/metronome/releases/tag/v0.3.2)).
+
+# Issues Fixed in DC/OS 1.9.9
+
+- COPS-2041/DCOS-21467 - DC/OS UI: Fixed labels reducer to accept empty strings.
+- COPS-3180 - Fixed odd exit behavior for tasks that use a volume mount (via rexray). [enterprise type="inline" size="small" /]
+- DCOS-14199 - Consolidated the Exhibitor bootstrapping shortcut by atomically reading and writing the ZooKeeper PID file. [enterprise type="inline" size="small" /]
+- DCOS-21557/DCOS_OSS-2367 - Updated cURL to version 7.59.
+- DCOS-22399/DCOS-22346 - Added privacy policy link to the login modal.
+
+# Notable Changes in DC/OS 1.9.9
+
+- DCOS-22171/DCOS-22184 - Admin Router DNS entry TTL is overwritten with 5 seconds.
+- DCOS_OSS-847 - Customized `ip_detect_public_filename` in config.yaml.
```

Comment:
We should make clear how this is relevant to users

Reply:
@klueska ^^